src/Router/Router.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Login from "../component/Login";
+import Admin from "../component/Admindashboard";
+import User from "../component/Userdashboard";
+import SignUp from "../component/Signup";
+import AddPoll from '../component/AddPoll';
+import UserDetails from '../component/UserDetails';
+import EditPoll from "../component/EditPoll";
+
+function Router() {
+  return (
+    <Routes>
+      <Route path="/" element={<Login />} />
+      <Route path="/signup" element={<SignUp />} />
+      <Route path="/admin" element={<Admin />} />
```

Comment:
if this is dashboard route , you can name this dashboard instead of admin
you can also pass dynamic route here for switching to admin dashboard and user dashboard.